CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# TutorialKit Contributing Guide
+
+Hi! We are really excited that you are interested in contributing to TutorialKit. Before submitting your contribution, please make sure to take a moment and read through the following guide:
+
+## Repo Setup
+
+The TutorialKit repo is a monorepo using pnpm workspaces. The package manager used to install and link dependencies must be [pnpm](https://pnpm.io/).
+
+To develop and test packages:
+
+1. Run `pnpm install` in project's root folder
+
+2. Run `pnpm run build` to build sources
+
+3. Run `pnpm run dev` to build sources in watch mode
```

Comment:
This works fine for most of the packages. Changes of `packages/components/react` are showing up on `packages/template` randomly.

Though sometimes `tsc` builds stop completely working and `scripts/clean.sh` is required. Personally I would rather remove TS project references as they cause hard to debug issues in this project.

Reply:
> Without project references the DX of navigating the code in this mono-repo becomes much worse.

You mean navigating in IDE by clicking the lines? That doesn't required project references, just `paths`.

Reply:
Don't `paths` need to duplicate the export logic from the package?

I might be remembering it wrong.

Maybe there's a better way to configure all this than by using project references. I would be tempted to keep it though because this is what we use in other repositories.

Reply:
Yep, it does need: https://github.com/vitest-dev/vitest/blob/32c0d99b3cd03df5570c6886b42498aa6a378fa2/tsconfig.base.json#L7-L34

Reply:
Ah ok, thanks for checking. Maybe we could move to that then, it doesn't seem so bad.

I think we should add that as an item to discuss to make sure everyone is on board with the idea.